src/components/Avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen } from "@testing-library/react"
+import { Avatar, AvatarSize } from "./Avatar"
```

Comment:
💡 What do you think of importing modules from the index? This would have the benefit of guaranteeing that we've actually exposed them as intended. Thoughts? (same for stories)

```suggestion
import { Avatar, AvatarSize } from "."
```
> In this can you may want to split up `Avatar` and `AvatarSize` as you may not intend to expose `AvatarSize`—though I don't see the harm. Could be as useful to the end-user as it was to you here. 🤷

Reply:
Wouldn't this create a circular refernce?